demo-shell/src/app/app.module.ts

```diff
@@ -199,8 +196,6 @@ registerLocaleData(localeSv);
             provide: HTTP_INTERCEPTORS, useClass:
             AuthBearerInterceptor, multi: true
         },
-        { provide: PROCESS_FILTERS_SERVICE_TOKEN, useClass: UserPreferenceCloudService },
```

Comment:
The reason behind this line was :
For enterprise: We should call actual preference services to get task/process filters.
For the community: We should use local storage to get task/process filters.

Is it not required anymore?
Can you please check once?

Reply:
When adding providers, specially with token injection, it's worth knowing that the parent provider overwrites a child provider. Because of this, the app.module.ts will overwrite all tokens of that kind making the community part also use the UserPreferenceService.

As it stands with this PR, it always makes use of the UserPreferenceService except for when it's overwritten in the community module.